docs/guide-de/intro-yii.md

```diff
@@ -0,0 +1,61 @@
+ist es besonders geeignet für die Entwicklung komplexer Anwendungen wie Online-Portalen, Foren, Content-Management-Systemen (CMS),
+E-Commerce-Projekten, RESTful Web Services und vieler mehr.
+
+
+Wie unterscheidet sich Yii von anderen Frameworks?
+--------------------------------------------------
+
+Wenn Sie bereits mit einem anderen Framework vertraut sind, wird es Sie vermutlich interessieren, wie Yii im Vergleich dazu abschneidet:
+
+- Wie die meisten PHP-Frameworks implementiert Yii das MVC (Model-View-Controller) Architekturpattern und fördert die
+  Organisation von Code, der auf diesem Muster basiert.
+- Yii vertritt die Philosophie, dass Code auf eine einfache und dennoch elegante Weise geschrieben werden sollte. Yii wird nie versuchen
+  Strukturen zu verkomplizieren, nur um ein bestimmtes Design-Pattern strikt einzuhalten.
+- Yii ist ein Full-Stack-Framework mit vielen bewährten und sofort einsatzbereiten Funktionen: Query-Builder
+  und ActiveRecord für relationale und NoSQL-Datenbanken; RESTful API-Entwicklungsunterstützung; mehrschichtige
+  Caching-Unterstützung; und mehr.
+- Yii ist sehr gut erweiterbar. Sie können fast jeden Teil des Kerncodes anpassen oder ersetzen. Sie können außerdem
+  die solide Erweiterungsarchitektur von Yii nutzen, um wiederverwendbare Erweiterungen zu verwenden oder zu entwickeln.
+- Hohe Leistung ist immer ein vorrangiges Ziel von Yii.
+
+Yii ist keine One-Man-Show, sondern wird von einem [starken Kern-Entwicklerteam](http://www.yiiframework.com/team/) sowie einer großen Community unterstützt,
+die ständig zur Entwicklung von Yii beitragen. Das Yii-Entwicklerteam
+behält die neuesten Webentwicklungstrends sowie die besten Praktiken und Funktionen im Auge,
+die in anderen Frameworks und Projekten zu finden sind. Die relevantesten Best Practices und Features, die anderswo zu finden sind, werden regelmäßig
+in das Kernframework integriert und über einfache und elegante Schnittstellen zugänglich gemacht.
+
+
+Yii Versionen
+-------------
+
+Yii steht derzeit in zwei Major-Versionen zur Verfügung: 1.1 und 2.0. Version 1.1 ist die alte Generation und befindet sich nun im Wartungsmodus.
+Version 2.0 ist eine komplette Neufassung von Yii, die die neuesten Entwicklungen,
+Technologien und Protokolle aufgreift, einschließlich Composer, PSR, Namespaces, Traits usw.
+Version 2.0 ist die aktuelle stabile Version des Frameworks und wird in den nächsten Jahren Hauptfokus der Entwicklung sein.
+In diesem Handbuch geht es hauptsächlich um die Version 2.0.
+
+
+Anforderungen und Voraussetzungen
+---------------------------------
+
+Yii 2.0 erfordert PHP 5.4.0 oder höher und läuft am besten mit der neuesten Version von PHP 7. Detailliertere Informationen über die
+Anforderungen einzelner Funktionen, können Sie erhalten, indem Sie den "requirements checker" ausführen, der in jeder Yii-Installation enthalten ist.
```

Comment:
`Funktionen, können Sie erhalten, indem` => `Funktionen erhalten Sie, indem`
Das Komma nach "Funktionen" ist in jedem Fall nicht korrekt.